### DISCUSSION TITLE
Apple Silicon support

### DISCUSSION THREAD

**Discussion by jacobo3d** [mitsuba-renderer/mitsuba3#180] [General]
Is Apple Silicon/Metal backend support planned? If so, is there an ETA? Thanks!

**Comment by wjakob** [comment]
We are thinking about it. There is no ETA, it could be on the timescale of months, years, or ultimately turn out to be impractical. Please let us know if you want to help out with this. In the meantime, you can use the LLVM backend on Macs, which uses SIMD/Neon and the latest version of Embree with M1-specific optimizations.

**Reply by jacobo3d** [reply]
Thanks for the info. LLVM will do for the time being, I was mainly curious about the roadmap regarding macOS
